pkg/storaged/stratis-panel.jsx

```diff
@@ -159,16 +174,28 @@ export function create_stratis_pool(client) {
                 return prepare_available_spaces(client, vals.disks).then(function (paths) {
                     const devs = paths.map(p => decode_filename(client.blocks[p].PreferredDevice));
 
-                    function create(key_desc) {
-                        return client.stratis_create_pool(vals.name, devs, key_desc).then(std_reply);
+                    function create(key_desc, adv) {
+                        let clevis_info = null;
+                        if (adv)
+                            clevis_info = ["tang", JSON.stringify({ url: vals.tang_url, adv })];
+                        return client.stratis_create_pool(vals.name, devs, key_desc, clevis_info).then(std_reply);
+                    }
+
+                    function create2(adv) {
+                        if (vals.encrypt_pass.on) {
+                            return get_unused_keydesc(client, vals.name)
+                                    .then(keydesc => with_stored_passphrase(client, keydesc, vals.passphrase,
+                                                                            () => create(keydesc, adv)));
+                        } else {
+                            return create(false, adv);
+                        }
                     }
 
-                    if (vals.encrypt.on) {
-                        return get_unused_keydesc(client, vals.name)
-                                .then(keydesc => with_stored_passphrase(client, keydesc, vals.passphrase,
-                                                                        () => create(keydesc)));
+                    if (vals.encrypt_tang && vals.encrypt_tang.on) {
+                        return get_tang_adv(vals.tang_url)
+                                .then(adv => confirm_tang_trust(vals.tang_url, adv, () => create2(adv)));
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18818-20230601-113309-0e7aeb29-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)